src/ShopBundle/objects/WebModules/MTShopBasketCore/MTShopBasketCoreEndpoint.class.php

```diff
@@ -1096,6 +1096,13 @@ public function GetHtmlHeadIncludes()
         return $aIncludes;
     }
 
+    public function GetPostRenderVariables(): array
+    {
+        $variables = parent::GetPostRenderVariables();
+        $variables['basketUrl'] = $this->getShopService()->getBasketLink(false);
+        return $variables;
```

Comment:
Nothing big, but the symfony conding standard want to have a blank line before a return.

Quote:
`Add a blank line before return statements, unless the return is alone inside a statement-group (like an if statement);`